### ISSUE TITLE
Any issue or comment is welcome!

### ISSUE THREAD

**Issue by skyforcetw** [skyforcetw/Garminuino#1]
Please write down your comment, or just talking!

**Comment by gary52** [comment]
Hello I just got my garminHUD to try with this. 2 issues i have so far. 1. In 0.1.5 release, nothing happen or show the bluetooth device menu when i hit "scan bluetooth".
2. I rolled back to 0.1.4. This time i got the garminHUD to pair. However it stuck showing OK, but nothing else after i started a navigation with my google maps.
(I am using a tmobile galaxy s8)

**Comment by Yan1980** [comment]
Hello,
No issues to report whatsoever and thank you for the great job.
I was going to dispose of my Garmin hud, but your app brought it back to life.
I was wondering since Waze is using Google maps, do you believe that your app can be modified so that Garmin hud can receive info from Waze?

**Comment by skyforcetw** [comment]
> Hello,
> No issues to report whatsoever and thank you for the great job.
> I was going to dispose of my Garmin hud, but your app brought it back to life.
> I was wondering since Waze is using Google maps, do you believe that your app can be modified so that Garmin hud can receive info from Waze?

Hi.

Unfotunally, Waze don't use "Notification" to hint the route, so it's need more work to make GarminHUD with Waze.

skyforce

**Comment by Yan1980** [comment]
Hello skyforcetw ,

Installed app ver. 0.5.4 on galaxy tab s2 android 7.0 and managed to connect to Garmin HUD.
Google maps ver. 10.29.1 with all notifications enabled. 
Garmin HUD shows speed and time , but there is no routing information from Google maps shown on HUD. Tested 3 buttons in debug window and arrows appear on HUD.
Notifications also appear in notification window of the tablet, but not on HUD.
On the main screen "GARMIN HUD" and "Notification Capture" are activated and "Gmap navigation Capture" is not activated. When driving updated speed and time are shown on HUD, but no routing information. In notification window of the app only  messages with updated speed are shown.
Hope to hear your thoughts.
Many thanks
P.S. in addition I tried to install app ver. 0.5.4. on car android device running on Android 5.1 , but during install error message regarding parsing appeared and installation terminated.

**Comment by skyforcetw** [comment]
Hi

Did you set the notification access?
https://github.com/skyforcetw/Garminuino/blob/master/GoogleMaps_HUD/FIRST_USE.md

**Comment by Yan1980** [comment]
Yes Sir, Notification access is enabled.  Kindest regards,  Yan1980








 -------- Оригинално писмо --------

От: skyforce notifications@github.com

Относно: Re: [skyforcetw/Garminuino] Any issue or comment is welcome! (#1)

До: skyforcetw/Garminuino  

Изпратено на: 17.11.2019 14:46



 
 
  
Hi  
  
Did you set the notification access?
  https://github.com/skyforcetw/Garminuino/blob/master/GoogleMaps_HUD/FIRST_USE.md

**Comment by skyforcetw** [comment]
> Yes Sir, Notification access is enabled. Kindest regards, Yan1980
> […](#)
> -------- Оригинално писмо -------- От: skyforce notifications@github.com Относно: Re: [skyforcetw/Garminuino] Any issue or comment is welcome! (#1) До: skyforcetw/Garminuino Изпратено на: 17.11.2019 14:46 Hi Did you set the notification access? https://github.com/skyforcetw/Garminuino/blob/master/GoogleMaps_HUD/FIRST_USE.md — You are receiving this because you commented. Reply to this email directly, view it on GitHub , or unsubscribe .

Please try this one:
https://github.com/skyforcetw/Garminuino/releases/download/0.5.5/gmaps_hud-release_v0.5.5.apk

I add some debug message, if no gmap's notify capture, it can show result:

"Notify parsing failed.";
"No gmaps' notification found!?!?"
"No notification found!?!?"

Please check the message you got.

**Comment by ObsidianVoice** [comment]
Thx for your great work! Just bought a GarminHUD because of your app! Is there a possibiliy to add an autostart and autoend to your application?

**Comment by skyforcetw** [comment]
> Thx for your great work! Just bought a GarminHUD because of your app! Is there a possibiliy to add an autostart and autoend to your application?

My pleasure ^^

Can you describe a bit about autostart & autoend?

**Comment by greymda** [comment]
i've installed it on my Android phone, and i cant enable:
1. Gmap navigation capture
2. Traffic & Lane Detect (Need Screenshot)

anything i'm missing obvious here?

**Comment by skyforcetw** [comment]
These two function are not stable, so I remove it, sorry!

**Comment by greymda** [comment]
> These two function are not stable, so I remove it, sorry!

got it! but the project is still working? if yes, Android 5 supported?

**Comment by seriojel** [comment]
hey @skyforcetw 
does it fetch current speed limit of the area and shows it on the HUD? like  the original Garmin app?

**Comment by skyforcetw** [comment]
No, that need extra work for it.

**Comment by seriojel** [comment]
> No, that need extra work for it.

@skyforcetw it would be so great and useful! i hope you can do it!
